dcos/dcos-cluster-one-master.json

```diff
@@ -248,7 +248,7 @@
       "type": "string"
     }, 
     "linuxVersion": {
-      "defaultValue": "16.04.201605161", 
+      "defaultValue": "latest", 
```

Comment:
We pin to a specific version because of words introduced upstream that caused an outage.